test/IRGen/builtin_math.swift

```diff
@@ -25,15 +25,29 @@ public func test2(f : Double) -> Double {
 // we want sqrt(negative) to be defined to be NaN for IEEE 754 conformance.
 
 // CHECK-LABEL: define {{.*}}test3
-// CHECK-NOT: call double @llvm.sqrt.f64
+// CHECK: call double @sqrt
 
 public func test3(d : Double) -> Double {
   return sqrt(d)
 }
 
 // CHECK-LABEL: define {{.*}}test4
-// CHECK-NOT: call float @llvm.sqrt.f32
+// CHECK: call float @sqrtf
 
 public func test4(f : Float) -> Float {
   return sqrt(f)
 }
+
+// CHECK-LABEL: define {{.*}}test5
+// CHECK: ret float 2
+
+public func test5( ) -> Float {
+  return sqrt(4)
+}
+
+// CHECK-LABEL: define {{.*}}test6
+// CHECK: ret double 2
+
+public func test6( ) -> Double {
+  return sqrt(4)
+}
```

Comment:
I think the earlier CHECK-NOT llvm.sqrt probably covers us, but I can definitely add one.